src/Services/GTT/GTT.Api/local.settings.json

```diff
@@ -3,7 +3,9 @@
   "Values": {
     "AzureWebJobsStorage": "UseDevelopmentStorage=true",
     "FUNCTIONS_WORKER_RUNTIME": "dotnet-isolated",
-    "SqlOptions__SqlConnectionString": "Data Source=tcp:alexdev.database.windows.net;Initial Catalog=alex;Persist Security Info=False;Encrypt=True;TrustServerCertificate=False;Connection Timeout=30;User ID=ffffff;Password=ffffff;",
-    "SqlOptions__SqlUseAccessToken": "false"
+    "SqlOptions__SqlConnectionString": "Data Source=VPHUDUC;Initial Catalog= GTT; database = GTT; User ID=sa;Password=123456",
+    "SqlOptions__SqlUseAccessToken": "false",
+    "AzureWebJobsStorageImage": "fffff",
```

Comment:
Can we group options for uploading image using prefix like UploadImageOptions__properties...something like SqlOptions